web/src/emoji_picker.ts

```diff
@@ -529,13 +593,13 @@ function process_keypress(e) {
         e.preventDefault();
         e.stopPropagation();
 
-        const $emoji_filter = $(".emoji-popover-filter");
+        const $emoji_filter = $<HTMLInputElement>(".emoji-popover-filter");
         const old_query = $emoji_filter.val();
         let new_query = "";
 
         if (pressed_key === 8) {
             // Handles Backspace.
-            new_query = old_query.slice(0, -1);
+            new_query = old_query?.slice(0, -1) ?? "";
```

Comment:
Shouldn't this be an `assert`( ie .```old_query ! == undefined```)? The reason for `undefined` is `.val()` function.